src/database/database.cpp

```diff
@@ -56,7 +56,14 @@ aos::Error Database::Init(const std::string& workDir, const MigrationConfig& mig
         CreateTables();
 
         mMigration.emplace(*mSession, migration.mMigrationPath, migration.mMergedMigrationPath);
-        mMigration->MigrateToVersion(cVersion);
+
+        CreateMigrationData(migration);
+        mMigration->MigrateToVersion(GetVersion());
+        DropMigrationData();
+    } catch (const Poco::Exception& e) {
```

Comment:
Please use unified version:
```suggestion
catch (const std::exception& e) {
        return utils::ToAosError(e);
    }
```

Reply:
done